plugins/Initial/src/Initial.ts

```diff
@@ -34,6 +36,12 @@ class InitialPluginInstance {
         l.attach(Downgraded)
         return !isEqual(l.value, this.getInitial(l.path))
     }
+    set = (l: StateMethods<StateValueAtRoot>) => {
+        this.initialState = l.value;
```

Comment:
This implementation will be valid only for when `l.path == []`. I believe set and reset should handle setting and resetting nested objects, like set for the `Hookstate.set` does. Have a look how `this.getInitial(l.path)` works and set should do similar to `this.setInitial(l.path)`.

Reply:
ahh good point. Will do.